tools/c7n_mailer/c7n_mailer/deploy.py

```diff
@@ -38,9 +38,19 @@ def dispatch(event, context):
 """
 
 
+def hack_ruamel():
+    # The way ruamel is installed it has no __init__.py(!). import finds it(!!)
+    # but imp.find_package doesn't(!!!). Hack it!
```

Comment:
importlib.import_module does find it

Reply:
Okay, lemme see what I can do ...

```
>>> ruamel = importlib.import_module('ruamel')
>>> ruamel.__file__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute '__file__'
```

Reply:
Looks like this means adding support for [namespace packages](https://www.python.org/dev/peps/pep-0420/) to `PythonPackageArchive`. Will work something up ...

Reply:
>>> importlib.import_module('ruamel').__path__
['/Users/xyz/projects/custodian/lib/python2.7/site-packages/ruamel']

Reply:
&rarr; https://github.com/capitalone/cloud-custodian/pull/1677, will update here